task-10/js/app.js

```diff
@@ -0,0 +1,105 @@
+// Section 1
+
+console.log(1 + 2);
+console.log("apple" + "orange");
+console.log(1 + 2 + "apple");
+console.log("apple" + 1 + 2);
+console.log(1 + true);
+console.log(0 == false);
+console.log(1 === true);
+console.log(2 == "2");
+
+// Section 2
+
+// 1
+
+cricketPlayers = ["Ashok", "Harish", "Dinesh", "Maamathi", "Kavya", "Ranjith", "Charan", "Hareesh", "Mathi", "Thendral", "Pavithra"]
+
+console.log(cricketPlayers);
+
+// 2
+
+console.log(cricketPlayers.shift())
+
+// 3
+
+console.log(cricketPlayers.length)
+
+// 4
+
+cricketPlayers.push("Ashok")
+console.log(cricketPlayers)
+
+// 5
+
+cricketPlayers.sort(function (a, b) {
+    return a.toLowerCase() - b.toLowerCase();
+});
+console.log(cricketPlayers);
+
+// 6
+
+cricketPlayersWithJerseyNumber = cricketPlayers.map((value) => value + " - " + Math.round(Math.random() * 100))
+console.log(cricketPlayersWithJerseyNumber);
+
+// 7
+
+cricketPlayersUpperCase = cricketPlayersWithJerseyNumber.map(value => value.toUpperCase())
+console.log(cricketPlayersUpperCase);
+
+// Section 3
+
+// 1 - Generate Numbers from 1 to 100
+
+function generateNumbers(start, end) {
+    let arr = []
```

Comment:
We only want to display numbers right ? Why do we store them in an array and return them ? but this is nice but not the requirement